test/integration/targets/iosxr_logging/tasks/cli.yaml

```diff
@@ -20,3 +20,6 @@
   with_first_found: "{{ test_items }}"
   loop_control:
     loop_var: test_case_to_run
+
```

Comment:
Might be worth adding a comment here to say why this is needed

Reply:
Yes